syntax = "proto3";
package communication_objects;

message AgentMetrics {
    float steps_per_episode = 1;
    float step_count = 2;
    float temp_done_counter = 3;
    float done_count = 4;
    float reward_per_step = 5;
    float reward_per_episode = 6;
    float reward_total = 7;
}